src/main/java/seedu/address/model/ClaimsBook.java

```diff
@@ -0,0 +1,121 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import javafx.collections.ObservableList;
+import seedu.address.model.claim.Claim;
+import seedu.address.model.claim.UniqueClaimsList;
+
+/**
+ * Wraps all data at the claims-book level
+ * Duplicates are not allowed (by .isSameClaim comparison)
+ */
+public class ClaimsBook implements ReadOnlyClaimsBook {
```

Comment:
Do we need a claimsbook? Can the UniqueClaimsList be stored under AddressBook with AddressBook as its wrapper? ie. AddressBook contains UniquePersonsList, UniqueClaimsList, UniqueIncomeList, etc.